material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -2014,3 +2015,148 @@ p.quick-ref-title {
   height: unset;
   padding: 1em 2em;
 }
+
+/* Cookie Consent Styling */
+.md-consent__overlay {
+  width: calc(97% - 1px);
+  z-index: 200;
+}
+
+.md-consent__inner {
+  z-index: 999;
+  background-color: transparent;
+}
+
+form.md-consent__form.md-grid.md-typeset {
+  background-color: var(--md-default-bg-color);
+  border: var(--border-grid);
+  display: flex;
+  padding: 1.5rem;
+  border-radius: 8px;
+  max-width: 90%;
+  margin: 2em auto;
+}
+
+.cookie-logo {
+  width: 11em;
+  display: flex;
+  justify-content: center;
+  margin: auto;
+}
+
+.cookie-logo img {
+  width: 4.5em;
+}
+
+.cookie-content {
+  flex: 1;
+  margin: 0 1em auto;
+  padding-right: 1em;
+}
+
+.cookie-content h4 {
+  margin-top: 0;
+  margin-bottom: 0.5em;
+  font-size: 15px;
+}
+
+.cookie-content p {
+  margin: 0;
+  font-size: 15px;
+  color: var(--light-transparent-60);
+  padding-bottom: 1em;
+  border-bottom: solid 2px var(--light-transparent-15);
+}
+
+.cookie-buttons {
+  margin-bottom: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.cookie-buttons .md-button {
+  margin: 0 2em 0;
+  width: 100%;
+  padding: 10px 45px;
+  text-align: center;
+  color: var(--light);
+}
+
+button.md-button.md-button--primary {
+  border: solid var(--border-width) var(--teal-light);
+  transition: background-color 0.3s ease-in-out;
+}
+
+button.md-button.md-button--primary:hover {
+  color: var(--light);
+}
+
+button.md-button.md-button--secondary {
+  background-color: var(--pink-dark-transparent-15);
+  border: solid var(--border-width) var(--pink-dark);
+}
+
+button.md-button.md-button--secondary:hover {
+  border: solid var(--border-width) var(--pink-dark);
+  background-color: var(--pink-dark);
+  color: var(--light);
+}
+
+.md-consent__controls label.md-button {
+  border: solid var(--border-width) var(--teal-light);
+  transition: background-color 0.3s ease-in-out;
+}
+
+.md-consent__controls label.md-button:hover {
+  color: var(--light);
+}
+
+.md-consent__settings {
+  margin: 0 1em auto;
+  animation: consent 0.5s cubic-bezier(0.1, 0.7, 0.1, 1) both;
+}
+
+ul.task-list {
+  margin: 0;
+  padding-top: 1em;
+}
+
+/* Cookie Consent Styling for Smaller Screens  */
+@media screen and (max-width: 768px) {
+  form.md-consent__form.md-grid.md-typeset {
+    flex-direction: column;
+    padding: 1rem;
+    max-width: 95%;
+  }
+
+  .cookie-logo img {
+    width: 3em;
+    margin: 1em auto;
+  }
+
+  .cookie-content h4 {
+    margin: 0 1em auto;
+  }
+
+  .cookie-content {
+    text-align: center;
+    margin: 0 auto;
+    padding-top: 0.5em;
+  }
+
+  .cookie-buttons {
+    gap: 0.75rem;
```

Comment:
```suggestion
    gap: 0.75rem;
    margin-top: 1em;
```

Reply:
Without the margin, it looks like this:

<img width="247" alt="Screenshot 2025-02-19 at 2 47 43 PM" src="https://github.com/user-attachments/assets/a0b14f3c-5a88-4a7b-98cf-040ddbda81f5" />

So let's give it some space between the buttons and the border/divider